_layouts/rnative-lesson.html

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+fw: true
+---
+<!-- ^idk what this does but im just putting it -->
+
+<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+
+<div class="rnative-content-container">
+    <div>
+        <div class="rnative-content-card">
+            <h1 class="title text-2x no-margin">{{page.title}}</h1>
+            <p>
+                {{page.desc}}
+            </p>
+            <hr class="divider-rnative" />
+            <ul class="list-unstyled text-15x">
+                {% if page.github_link %}
+                    <li>
+                        <a class="rnative-link" href="{{page.github_link}}" target="_blank" rel="noopener noreferrer">
+                            <span class="fa fa-fw fa-chalkboard-teacher"></span> Link to Github
```

Comment:
The icon for this can be `fab fa-fw fa-github`; I'm also debating if the copy could be better (e.g. Link to GitHub Repository, or if we want to frame it as notes, etc.)